platform/src/main/java/org/hillview/table/PrivacySchema.java

```diff
@@ -23,6 +26,17 @@ public PrivacyMetadata get(String colName) {
         return metadata.get(colName);
     }
 
+    public PrivacyMetadata get(String[] colNames) {
```

Comment:
Can you please add a test with the private metadata that uses multiple columns?
Including serialization/deserialization, etc.